libs/classifiers/classifier.py

```diff
@@ -1,17 +1,24 @@
+import logging
+
+logging.getLogger().setLevel(logging.INFO)
+
 class Classifier:
 
```

Comment:
Could you add the docstring for the class?

Reply:
Sure, thanks.